zebra-network/src/peer_set/initialize.rs

```diff
@@ -142,7 +154,9 @@ where
         );
     }
 
+    info!("checkpoint: {}", line!());
     let listen_guard = tokio::spawn(listen(config.listen_addr, listener, peerset_tx.clone()));
```

Comment:
Should this be spawn_blocking, because the listener bind has a long timeout on some platforms?